as_perimenoun_oi_gynaikes.ass

```diff
@@ -173,13 +173,13 @@ Dialogue: 0,0:10:46.64,0:10:49.22,Default,,0,0,0,,{\i1}Don't speak, don't speak.
 Dialogue: 0,0:10:49.22,0:10:49.98,Default,,0,0,0,,{\i1}...only with words...{\i0}
 Dialogue: 0,0:10:49.98,0:10:51.98,Default,,0,0,0,,Panos?
 Dialogue: 0,0:10:53.84,0:10:54.92,Default,,0,0,0,,Panos?
-Dialogue: 0,0:10:56.32,0:10:58.44,Default,,0,0,0,,She shows herself, right?
+Dialogue: 0,0:10:56.32,0:10:58.44,Default,,0,0,0,,She shows up, right?
 Dialogue: 0,0:10:58.44,0:10:59.84,Default,,0,0,0,,And she tells me, let's go...
 Dialogue: 0,0:10:59.84,0:11:01.84,Default,,0,0,0,,...and it's like my life restarts.
 Dialogue: 0,0:11:02.98,0:11:04.48,Default,,0,0,0,,Panos...
 Dialogue: 0,0:11:04.48,0:11:07.44,Default,,0,0,0,,...I'm picturing the girl coming \Nand telling you "let's go"...
 Dialogue: 0,0:11:07.44,0:11:09.30,Default,,0,0,0,,...do you put her in the truck \Nand you go?
-Dialogue: 0,0:11:09.30,0:11:12.20,Default,,0,0,0,,And she gets black from the filth \Nfrom head to toe?
+Dialogue: 0,0:11:09.30,0:11:12.20,Default,,0,0,0,,And it gets black from the filth \Nfrom head to toe?

```

Comment:
This is wrong, he refers to the woman, not the car.

Reply:
https://twitter.com/apogmixalis/status/906874812269821952

Πάνο; Σκέφτομαι πως έρχεται το κορίτσι και σου λέει πάμε.
Τη βάζεις στο φορτηγό και πάτε; και γίνεται μαύρο από πάνω μέχρι κάτω;

Μαύρο όχι, μαύρη... είναι περίεργη η αλλαγή του φύλου της αντωνυμίας συνεχώς (ουδέτερο, θηλυκό, ουδέτερο). Το ότι γίνεται το φορτηγό μαύρο σημαίνει ότι γίνονται 'ακολασίες' μέσα του.

Reply:
"Μαύρο" refers to "το κορίτσι" (both neuter, i.e. "το μαύρο κορίτσι"). You are reading this wrong.

Reply:
Think about it a bit and it will make sense. The girl will get into the dirty ironsmith truck and get dirty herself. "Mαύρο από πάνω μέχρι κάτω" is a common expression for getting physically dirty.